server/app/views/applicant/ProgramCardsSectionParamsFactory.java

```diff
@@ -24,8 +29,32 @@ public ProgramCardsSectionParamsFactory(ApplicantRoutes applicantRoutes) {
     this.applicantRoutes = checkNotNull(applicantRoutes);
   }
 
+  public ProgramSectionParams getSection(
+      Messages messages,
+      Optional<MessageKey> title,
```

Comment:
**question:** In what cases would the title be unset? Is that for the "get started" / pre screener section if it applies?

Reply:
I don't actually know, I was just following the original implementation. If we can figure it out maybe I can add a test for it.

Reply:
It looks like that's what it is! https://docs.civiform.us/user-manual/civiform-admin-guide/working-with-programs/set-pre-screener. There's some testing for it [here](https://github.com/civiform/civiform/blob/7d97c1ece46abd8b88d63500c413b45921ec7584/browser-test/src/applicant/applicant_application_index.test.ts#L166-L173), but it probably is worth adding here as well.